Server/bkr/server/kickstart.py

```diff
@@ -22,6 +23,8 @@
 from bkr.server.model.distrolibrary import split_osmajor_name_version
 from bkr.server.util import absolute_url
 
+from six.moves.urllib import parse as urlparse
```

Comment:
Could this be put up around line 17?

Reply:
I mean, the whole sorting is all over the place, I don't want to do it now. We will do it as soon as all Python 2 libraries are out + the six so we can use proper tooling w/ Python 3.